src/rgw/setup/radosgw_start

```diff
@@ -11,6 +11,6 @@ echo "Change RGW CWD to dir $RGW_CWD, to retain core files in there"
 cd $RGW_CWD
 
 set -o pipefail;
-/usr/bin/radosgw -f --name client.rgw-$INDEX -c $CONFIG_FILE --no-mon-config 2>1 \
- | gawk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }'  | tee $LOG_FILE
-exit $?
\ No newline at end of file
+/usr/bin/radosgw -f --name client.rgw-$INDEX -c $CONFIG_FILE --no-mon-config 2>&1 \
+ | while read line; do printf "%s %s\n" "$(date +'[%Y-%m-%d %H:%M:%S,%03N]')" "$line"; done | tee $LOG_FILE
```

Comment:
changing gawk to a loop and printing the logs also resolved the disappearing logs issue
@cdeshmukh , @sachitanands 
tested with custom build wit this changes

```
[root@ssc-vm-rhev4-2782 ~]# kubectl logs cortx-server-ssc-vm-rhev4-2782-7fb795ccf9-qwwnj -c cortx-rgw
Change RGW CWD to dir /etc/cortx/log/rgw/289486004938be404226e0fb94e6b70a/rgw_debug, to retain core files in there
[2022-06-24 07:11:38,646] motr[00009]:  7460   WARN  [ha/entrypoint.c:563:ha_entrypoint_client_fom_tick]  rlk_rc=-110
[2022-06-24 07:11:44,058] motr[00009]:  9410  ERROR  [dix/req.c:800:dix_idxop_meta_update_ast_cb]  All items are failed
[2022-06-24 07:11:44,061] motr[00009]:  c410  ERROR  [dix/req.c:800:dix_idxop_meta_update_ast_cb]  All items are failed
[2022-06-24 07:11:44,065] motr[00009]:  e410  ERROR  [dix/req.c:800:dix_idxop_meta_update_ast_cb]  All items are failed
[2022-06-24 07:11:44,069] motr[00009]:  a410  ERROR  [dix/req.c:800:dix_idxop_meta_update_ast_cb]  All items are failed
[2022-06-24 07:11:44,072] motr[00009]:  a410  ERROR  [dix/req.c:800:dix_idxop_meta_update_ast_cb]  All items are failed
[2022-06-24 07:13:16,143] motr[00009]:  2310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[2022-06-24 07:13:16,147] motr[00009]:   310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[2022-06-24 07:13:17,986] motr[00009]:  c310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[2022-06-24 07:13:18,107] motr[00009]:  4310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[2022-06-24 07:13:18,194] motr[00009]:  9310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[2022-06-24 07:13:19,248] motr[00009]:   310  ERROR  [cas/client.c:1177:cas_req_replied_ast]  <! rc=-2
[root@ssc-vm-rhev4-2782 ~]# 
```

Reply:
nice catch !!